### PR TITLE
update spanish and italian translate 11072023

### PR DESCRIPTION
**Pull request by johnfelipe**

### CODE REVIEW

File: frontend/src/locales/it.json [hunk 1]
```diff
@@ -2,54 +2,54 @@
...
-      "label": "Informazioni",
+      "label": "Circa",
       "repository": "GitHub",
       "docs": "Documentazione"
     },
     "licenses": {
-      "frontend": "Licenze (Frontend)",
-      "backend": "Licenze (Backend)",
+      "frontend": "Licenze (frontend)",
+      "backend": "Licenze (backend)",
```
Comment (KzuDemEvin) [inline frontend/src/locales/it.json:11]: Use `Backend` and `Frontend` instead of `backend` and `frontend`

**Review by KzuDemEvin — CHANGES_REQUESTED** [review]
Thank you for your improvements!
For a more consistent translation, both the `Backend` and the `Frontend` should be capitalized.

**Comment by johnfelipe** [comment]
Changed today

**Comment by KzuDemEvin** [comment]
Can you push the changes?

**Review by KzuDemEvin — APPROVED** [review]

**Review by SponsoredByPuma — APPROVED** [review]

**Pull request merged**